### PR TITLE
[DOCFIX] Improve Security doc

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Security.md [hunk 1]
```diff
@@ -9,71 +9,48 @@ priority: 1
...
-Authorization Service) is used to determine who is currently executing the process.
-
-When authentication is enabled, a login user for the component (master, worker, or client)
-can be obtained by following steps:
+### SIMPLE
 
-1. Login by configurable user. If property 'alluxio.security.login.username' is set by
-application, its value will be the login user.
-2. If its value is empty, login by OS account.
+When `alluxio.security.authentication.type=SIMPLE`, authentication is enabled.
```
Comment (gpang) [inline docs/en/Security.md]: This is not a complete sentence?

File: docs/en/Security.md [hunk 2]
```diff
@@ -9,71 +9,48 @@ priority: 1
...
+Before an Alluxio client accessing the service, this client retrieve the user information to report
```
Comment (gpang) [inline docs/en/Security.md]: "accessing" => "accesses"
"this client retrieve" => "the client retrieves"

File: docs/en/Security.md [hunk 3]
```diff
@@ -112,13 +89,13 @@ For example, the output of the shell command `ls -R` when authorization is enabl
 ### User group mapping
 
 When user is determined, the list of groups is determined by a group mapping service, configured by
-'alluxio.security.group.mapping.class'. The default implementation is 'alluxio.security.group
-.provider.ShellBasedUnixGroupsMapping', which executes the 'groups' shell
+`alluxio.security.group.mapping.class`. The default implementation is
+`alluxio.security.group.provider.ShellBasedUnixGroupsMapping`, which executes the 'groups' shell
```
Comment (gpang) [inline docs/en/Security.md]: `groups`

File: docs/en/Security.md [hunk 4]
```diff
@@ -183,7 +160,8 @@ The format of Alluxio audit log entry is shown in the table below.
 
 It is similar to the format of HDFS audit log [wiki](https://wiki.apache.org/hadoop/HowToConfigure).
 
-To enable Alluxio audit logging, you need to set the JVM property `alluxio.master.audit.logging.enabled` to true, see [Configuration settings](Configuration-Settings.html).
+To enable Alluxio audit logging, you need to set the JVM property
+`alluxio.master.audit.logging.enabled` to true, see [Configuration settings](Configuration-Settings.html).
```
Comment (gpang) [inline docs/en/Security.md]: `true`

File: docs/en/Security.md (hunk 1 above)
Comment (apc999) [reply docs/en/Security.md]: fixed

File: docs/en/Security.md (hunk 2 above)
Comment (apc999) [reply docs/en/Security.md]: fixed

**Comment by apc999** [comment]
@gpang  PTAL

**Pull request merged**